src/components/window-dimensions.tsx

```diff
@@ -0,0 +1,49 @@
+import { useState, useEffect } from 'react';
```

Comment:
Super useful! @jackblanc is this where we'd want to keep it? Not sure how a React project should be set up

Reply:
"it" as in this file. Which folder should it be in